lib/serialisers/json-0.6.js

```diff
@@ -20,17 +20,17 @@ module.exports = createClass({
       element: element.element,
     };
 
-    if (element.meta.length > 0) {
+    if (element._meta && element._meta.length > 0) {
       payload['meta'] = this.serialiseObject(element.meta);
     }
 
-    if (this[element.element + 'SerialiseAttributes']) {
-      var attributes = this[element.element + 'SerialiseAttributes'](element);
+    if (element.element === 'enum') {
```

Comment:
Since there is only custom serialise attributes for enum, it's faster to explicitly check instead of dynamically dispatch.